tests/testthat/_snaps/update-news.md

```diff
@@ -1,3 +1,37 @@
+# Can parse Co-authored-by
+
+    Code
+      extract_newsworthy_items(
+        "- blop\n-blip\n\nCo-authored-by: Person (<person@users.noreply.github.com>)")
```

Comment:
as one could store several bullets per commit message.
note fledge does not support several _types_ per commit message (and will probably never support a mix of bullets and types in a commit message, at least I hope so :sweat_smile: )